dockerfile.template

```diff
@@ -15,21 +15,23 @@ RUN apt-get -qq install -y --no-install-recommends \
 RUN wget -q https://bootstrap.pypa.io/get-pip.py
 RUN python2.7 get-pip.py
 
-RUN pip2 -q install --upgrade pip
-RUN pip2 -q install setuptools
-RUN pip2 -q install azure azure-cli
+RUN python2.7 -m pip -q install pip==9.0.3
```

Comment:
Do we know what runs with python 2.7? Are setuptools and azure-cli 2.7 only?  Ideally I'd prefer to never use it but if we can at least comment why that might be useful.

Reply:
AFAIK the pip2 stuff is required for az cli to work with the VSTS service but msftkube used python3. I may be mistaken as I wanted to not add both pythons as well but I saw this issue during the original creation of this file when it would fail to run once deployed.